src/index.js

```diff
@@ -28,26 +43,27 @@ module.exports = function purescriptLoader(source, map) {
     pscIde: false,
     pscIdeColors: webpackOptions.psc === 'psa' || query.psc === 'psa',
     pscIdeArgs: {},
+    pscPackage: false,
     bundleOutput: 'output/bundle.js',
     bundleNamespace: 'PS',
     bundle: false,
     warnings: true,
     output: 'output',
     src: [
       path.join('src', '**', '*.purs'),
-      path.join('bower_components', 'purescript-*', 'src', '**', '*.purs')
+      ...depsPaths(query.pscPackage)
```

Comment:
All right, [es2015 preset does include spread operator](https://babeljs.io/docs/plugins/transform-es2015-spread/) so the answer is no, we don't.